src/LibP4est_v2.jl

```diff
@@ -13063,6 +13063,8 @@ const P4EST_QCOORD_MAX = INT32_MAX
 
 const P4EST_QCOORD_1 = p4est_qcoord_t(1)
 
+P4EST_QCOORD_ABS(x) = (p4est_qcoord_t(labs))(Clong(x))
```

Comment:
What is `labs` (here and below) - and `llabs` below?

Reply:
You are right. I remove these functions again. We probably don't need them.